### ISSUE TITLE
Nothing works

### ISSUE THREAD

**Issue by HAMPERHAMPS** [face-hh/webx#211]
When I try using Bussin it just says this site doesn't exist. When I go to buss://dingle.it it also says the site doesn't exist.

**Comment by GStudiosX2** [comment]
What does it show when you do `ctrl + shift + s`?

**Comment by HAMPERHAMPS** [comment]
All it shows is this when I do that.
![image](https://github.com/user-attachments/assets/8ab81a6d-d160-4a7f-a4b0-acdf88cd5780)

**Comment by GStudiosX2** [comment]
> All it shows is this when I do that.
> ![image](https://github.com/user-attachments/assets/8ab81a6d-d160-4a7f-a4b0-acdf88cd5780)
> 

Try entering in the box `https://api.buss.lol`

**Comment by HAMPERHAMPS** [comment]
It's just blank. Is it a problem with my PC?
![image](https://github.com/user-attachments/assets/f3aff1ca-3ce9-4252-bf13-533045995241)

**Comment by GStudiosX2** [comment]
> It's just blank. Is it a problem with my PC? ![image](https://private-user-images.githubusercontent.com/149680144/350783907-f3aff1ca-3ce9-4252-bf13-533045995241.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE1ODMwNDksIm5iZiI6MTcyMTU4Mjc0OSwicGF0aCI6Ii8xNDk2ODAxNDQvMzUwNzgzOTA3LWYzYWZmMWNhLTNjZTktNDI1Mi1iZjEzLTUzMzA0NTk5NTI0MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyMVQxNzI1NDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZWFjYWJlOWZmNTUwNzQ1YWU3NzkzZDk5ZDEyODAzNDBkYTI2OWY1OWVjYTAwMjFlZTE2ZTJmZTNmNzFmNzgxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.hgIZgVXjO8p1ZB-SPZuq5ymMzkqVrGtggV418MbA7fU)

Type in the box `https://api.buss.lol` it shouldn't be empty though for me it was `api.buss.lol` but it needs to be `https://api.buss.lol`

**Comment by HAMPERHAMPS** [comment]
When I type that the text is blank like in the image I sent before. Is it doing that because some of my graphics drivers are outdated?

**Comment by GStudiosX2** [comment]
> When I type that the text is blank like in the image I sent before. Is it doing that because some of my graphics drivers are outdated?

Uh maybe? Or it's a bug in the windows version

**Comment by HAMPERHAMPS** [comment]
Oh so there's no way for me to fix it?

**Comment by GStudiosX2** [comment]
> Oh so there's no way for me to fix it?

I'm not entirely sure I haven't ran into this issue before you can try and upgrade your graphics drivers though

**Comment by inventionpro** [comment]
> All it shows is this when I do that. ![image](https://private-user-images.githubusercontent.com/149680144/350734493-8ab81a6d-d160-4a7f-a4b0-acdf88cd5780.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE5MjY3MDUsIm5iZiI6MTcyMTkyNjQwNSwicGF0aCI6Ii8xNDk2ODAxNDQvMzUwNzM0NDkzLThhYjgxYTZkLWQxNjAtNGE3Zi1hNGIwLWFjZGY4OGNkNTc4MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyNVQxNjUzMjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hODQ5NGQyODI5MThkZDhjYWM2MWU3YzM5MjgwZDBlMGMzZDM1N2EzMDNkMTc1NDFiYjQ1MTE4MjZiYmViMzU1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.KYbspEciRJVbyrcss5vbQ3pPULcwV-aHwR9cDJBFp1o)

Did you use the exe from the github or did you manually compile it?
If you just downloaded it, i would recommend compiling it yourself since that should fix most issues.

**Comment by HAMPERHAMPS** [comment]
Yeah I just used the exe. I will try compiling it and see if that works. Thanks!
